engine/installation/linux/docker-ee/suse.md

```diff
@@ -167,12 +168,30 @@ from the repository.
 1.  Use the following command to set up the **stable** repository, using the
     Docker EE repository URL you located in the [prerequisites](#prerequisites).
 
+    **x86_64**:
+
     ```bash
     $ sudo zypper addrepo \
         <DOCKER-EE-URL>/sles/12.3/x86_64/stable-{{ site.docker_ee_version }} \
         docker-ee-stable
     ```
 
+    **s390x**:
+
+    ```bash
+    $ sudo zypper addrepo \
+        <DOCKER-EE-URL>/sles/12.3/s390x/stable-{{ site.docker_ee_version }} \
+        docker-ee-stable
+    ```
+
+    **ppc64le**:
+
+    ```bash
+    $ sudo zypper addrepo \
+        <DOCKER-EE-URL>/sles/12.3/ppc64le/stable-{{ site.docker_ee_version }} \
```

Comment:
this probably needs to changed also to ppc64el, but because I can't see it to verify ping @dmp42

Reply:
`sles/12.3/ppc64le` is correct

@tophj-ibm you should ask on the slack chan to get an EE license (that would let you access / check urls).

Reply:
Doc accurately reflects reality in its current state.

Reply:
Doing some wider looking, it looks like this is split. debian based (debian/ubuntu) use `ppc64el` and redhat based (suse, centos, fedora, rhel) use `ppc64le`. (this isn't confusing at all 😕 ) so this one is correct, the ubuntu one is not.

Reply:
Wondering if we should use tabs for this section (just a nice to have; can be in a follow-up)